packages/obeam/obeam.0.1.3/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+synopsis: "A utility library for parsing BEAM format"
+maintainer: "yutopp <yutopp@gmail.com>"
+authors: ["yutopp <yutopp@gmail.com>" "amutake <amutake.s@gmail.com>"]
+license: "Boost License Version 1.0"
+homepage: "https://github.com/yutopp/obeam"
+bug-reports: "https://github.com/yutopp/obeam/issues"
+depends: [
+  "ocaml" {>= "4.07.0"}
```

Comment:
I think this used to work for earlier ocaml versions, are you using some newer feature that broke that compatibility?